lte/cloud/go/services/lte/obsidian/models/conversion.go

```diff
@@ -200,6 +200,9 @@ func (m *LteGateway) FromBackendModels(
 	}
 	sort.Strings(m.ConnectedEnodebSerials)
 
+	checkedInRecently := GatewayCheckedInRecently(orc8rModels.LastGatewayCheckInWasRecent(m.Status, m.Magmad))
+	m.CheckedInRecently = &checkedInRecently
```

Comment:
same here , you can save one variable 

```
m.CheckedInRecently = &GatewayCheckedInRecently(orc8rModels.LastGatewayCheckInWasRecent(m.Status, m.Magmad))

Reply:
Same problem here, I get a compile error if I don't create a variable before setting the pointer. I don't think Go has a more convenient way to do that.